wled00/file.cpp

```diff
@@ -34,7 +34,6 @@ File f;
 //wrapper to find out how long closing takes
 void closeFile() {
   DEBUGFS_PRINT(F("Close -> "));
-  uint32_t s = millis();
```

Comment:
This is needed if FS debugging is enabled (see 2 lines below). Rather than removing we should enclose it in an `#ifdef WLED_DEBUG_FS`